docker_services_cli/services.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+#
+# Docker-Services-CLI is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Services module."""
+
+import subprocess
+
+
+def services_down(filepath="docker_services_cli/docker-services.yml"):
+    """Stops the given services.
+
+    It does not requries the services. It stops containers and removes
+    containers, networks, volumes, and images created by `up`.
+    """
+    command = ["docker-compose", "--file", filepath, "down"]
+
+    subprocess.run(command, check=True)
+
+
+def services_up(services, filepath="docker_services_cli/docker-services.yml"):
+    """Start the given services up.
+
+    docker-compose is smart about not rebuilding an image if
+    there is no need to, so --build is not a slow default. In addition
```

Comment:
That is the point. Imagine that the image of some service defined in the `docker-compose.yaml` is a `Dockerfile`. In that case it would need to be built (the image). In the past, it was required to add `--build`, however, now *docker-compose* is smart enough to do "guess" it correctly. If we add said flag, it will try to build all and be a *slow default*.